testcontainers/src/core.rs

```diff
@@ -1,20 +1,15 @@
-pub(crate) use container::Docker;
-#[cfg(feature = "experimental")]
-pub(crate) use container_async::DockerAsync;
-
-pub use self::{
-    container::Container,
-    image::{ContainerState, ExecCommand, Host, Image, ImageArgs, Port, RunnableImage, WaitFor},
+pub use self::image::{
+    ContainerState, ExecCommand, Host, Image, ImageArgs, Port, RunnableImage, WaitFor,
 };
 
-#[cfg(feature = "experimental")]
-pub use self::container_async::ContainerAsync;
+pub use self::containers::*;
 
-mod container;
-#[cfg(feature = "experimental")]
-mod container_async;
-pub mod env;
 mod image;
 
+pub(crate) mod client;
+pub(crate) mod containers;
+pub(crate) mod env;
```

Comment:
I personally like to avoid using abbreviations for more clarity in code, but that is a matter of taste.